widgets/sidebar/sidebar.tsx

```diff
@@ -38,6 +39,9 @@ export const Sidebar = () => {
     setOpen(false);
```

Comment:
There is a lack of functionality here. The modal window for logout confirmation should remain open but disabled until the ajax request to logout completes successfully.
If the response fails, the modal window should remain open but enabled.